src/components/MajorFilter.jsx

```diff
@@ -40,6 +40,7 @@ function MajorFilter({ majors, onSelect, onDelete }) {
                 {isOpen ? <ArrowDropUpIcon /> : <ArrowDropDownIcon />}
               </IconButton>
             }
+            sx={{ maxWidth: 300 }}
```

Comment:
added to avoid this
<img width="873" alt="Screen Shot 2022-02-02 at 6 15 39 PM" src="https://user-images.githubusercontent.com/40483569/152270258-19b87949-4ec4-4ea6-b965-f462e93527da.png">

Reply:
Seems fine to me. There are some pretty long majors like "Health information and medical records technology"